ReactWindows/ReactNative/ReactInstanceManager.cs

```diff
@@ -373,7 +373,7 @@ public IReadOnlyList<IViewManager> CreateAllViewManagers(ReactContext reactConte
         private void RecreateReactContextInBackgroundFromBundleFile()
         {
             RecreateReactContextInBackground(
-                () => new ChakraJavaScriptExecutor(),
+                () => new NativeJavaScriptExecutor(),
```

Comment:
Can we do the switch to NativeJavaScriptExecutor in a separate review, after we get some numbers posted on startup times, and average times for certain operations over the bridge?